src/Config/Converters/LogLevelOptionsConverterFactory.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Azure.DataApiBuilder.Config.ObjectModel;
+
+namespace Azure.DataApiBuilder.Config.Converters;
+
+/// <summary>
+/// Defines how DAB reads and writes log level options
+/// </summary>
+internal class LogLevelOptionsConverterFactory : JsonConverterFactory
```

Comment:
We use a converter factory because it was made with the idea in mind that in the future, we can allow the user to change each of the different providers individually, and they would reside inside the `LogLevelOptions`. It also helps keep the pattern that was set by the other properties inside of runtime.